start.R

```diff
@@ -304,8 +304,12 @@ if ("--reset" %in% argv) {
   source("./config/default.cfg")
   cfg$gms$c_expname <- cfg$title
   cfg$gms$c_description <- substr(cfg$description, 1, 255)
+  lock_id <- gms::model_lock(timeout1 = 1)
```

Comment:
Minor thing: shouldn't the timeout be shorter? `--reset` is an interactive operation, no?